draft-lenders-dns-over-coap.md

```diff
@@ -319,10 +319,10 @@ URI configuration
 
 Considerations for Unencrypted Use
 ==================================
-- TBD
-- DTLS-transport should be used
-- Non-DTLS can have benefits: Blockwise-transfer for IEEE 802.15.4, additional
-  layer of caching, ...
+While not recommended, using DoC with unencrypted CoAP can also have its benefits in very
+constrained environments where encryption is not possible or necessary. Potential benefits of
```

Comment:
I've tried shoving parts of that change around:

```suggestion
While not recommended,
DoC can be without any encryption
(e.g., in very constrained environments where encryption is not possible or necessary).
It can also be used when lower layers provide secure communication between client and server.
In both cases,
potential benefits of
```

is an attempt to keep the statement about no-security-at-all (not recommended) separate from no-security-coap-sees (which still keeps the benefits). Not sure it is making things better; either version is an enhancement and covers the TBD items (except the add'l layer of caching, but that would also be provided by a DNS resolver).